02week/pigLatin.js

```diff
@@ -7,17 +7,97 @@ const rl = readline.createInterface({
   output: process.stdout
 });
 
+/*
+pig latin code plan
+
+0. Write a function to check for valid input
+  1. Check that input is a string
+  2. Check for other things such as
+
+isValidWord= (word) =>{
+
+}
+
+1. Write a function that takes in the word and returns the word as a lower case, whitespace-trimmed word
+
+caseSpace= (word) => {
+  word = word.toLowerCase().trim();
+}
+
+2. Write a function that checks for a vowel
+
+
+3. Write a function that attaches "yay" to the end of a string
 
-function pigLatin(word) {
 
-  // Your code here
+4. Write a function that:
+ 1. splits the string at the first vowel
+ 2. stores the first item and deletes it from the array
+ 3. pops that item to the end
+ 4. pops "ay" to the end
+ 5. joins the array as a string
+ 6. returns the string
 
+5. Write the parent code to organize pure codes
+*/
+
+
+const vowelStartYayEnd = (word) => {
+  //checks if anything in the array (vowels) is in the first position (first letter) of the word. This method returns -1 if it doesn't match anything in the array
+  if (["a", "e", "i", "o", "u"].indexOf(word[0]) > -1) {
```

Comment:
don't reassign the word parameter that your given, that's an unnecessary step just 
```javascript
return word + "yay"
```

Reply:
fixed